app/commands/exercise/update_hash_for_important_exercise_files.rb

```diff
@@ -0,0 +1,28 @@
+class Exercise
+  class UpdateHashForImportantExerciseFiles
```

Comment:
Maybe something like:

```suggestion
  class UpdateImportantFilesHashWithSolutions
```